CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [4.15.2](https://github.com/plivo/plivo-java/tree/v4.15.2) (2021-06-18)
+- Removed total_count field from meta data for list MDR response. getTotalCount() for list MDR will return 0.
+- Moved get() method to PowerPackLister
```

Comment:
Let's not move this in current minor release because get() was public. You can revert the code.